packages/hash/frontend/src/components/grid/grid.tsx

```diff
@@ -161,34 +162,11 @@ export const Grid = <T extends RowData>({
           : undefined,
     });
   };
-  const interactableCustomRenderers = useMemo<
-    DataEditorProps["customRenderers"]
-  >(() => {
-    return customRenderers?.map((customRenderer) => {
-      return {
-        ...customRenderer,
-        draw: (args, cell) =>
-          customRenderer.draw({ ...args, tableId: tableIdRef.current }, cell),
-        onClick: (args) => {
-          /** @todo investigate why `args` don't have `location` in it's type  */
-          const [col, row] = (args as unknown as { location: Item }).location;
 
-          const wasClickHandledByManager = InteractableManager.handleClick(
-            `${tableIdRef.current}-${col}-${row}`,
-            args,
-          );
-
-          if (wasClickHandledByManager) {
-            args.preventDefault();
-          } else {
-            customRenderer.onClick?.(args);
-          }
-
-          return undefined;
-        },
-      };
-    });
-  }, [customRenderers]);
+  const overriddenCustomRenderers = useMemo(
+    () => overrideCustomRenderers(customRenderers, tableIdRef.current),
```

Comment:
You should be passing the ref, not the value within it, as value may change without re-calling `useMemo`. Reading refs during render isn't safe as they're not updated until post-render, and `useMemo` is essentially the same as render. 

(i think it's currently the case that `tableIdRef` never changes, but if someone introduces that, this will break).